astropy/coordinates/builtin_frames/fk4.py

```diff
@@ -144,11 +144,11 @@ def fk4_e_terms(equinox):
         The equinox for which to compute the e-terms
     """
     # Constant of aberration at J2000
-    k = 0.0056932
+    k = 0.0056932  # in degrees (v_earth/c ~ 1e-4 rad ~ 0.0057 deg)
```

Comment:
For a source, we could cite Explanatory Supplement to the Astronomical Almanac (Seidelmann, 2005) which gives this exact value on p171.